notesserver/settings/devstack.py

```diff
@@ -1,12 +1,22 @@
-from notesserver.settings.logger import get_logger_config
+from notesserver.settings.logger import build_logging_config
 
 from .common import *
 
 DEBUG = True
 
 ALLOWED_HOSTS = ['*']
 
-LOGGING = get_logger_config(debug=DEBUG, dev_env=True, local_loglevel='DEBUG')
+
+LOG_SETTINGS = {
```

Comment:
I don't see how or where this is used. Can we kill it?  Why or why not?

Reply:
The devstack one is used here:
https://github.com/edx/devstack/blob/master/docker-compose.yml#L177-L196

It is my belief that it should be using the yaml config and should load a yaml from that repo instead to reduce the number of codepaths, but that is currently not the case.

Reply:
Good catch - this was a precursor to the broken out variables I am now using - removed.